src/core/metadata/mod.rs

```diff
@@ -84,6 +87,10 @@ impl WalletMetadata {
             ClaimFormatDesignation::JwtVcJson,
             ClaimFormatPayload::AlgValuesSupported(alg_values_supported.clone()),
         );
+        vp_formats_supported.insert(
```

Comment:
This is the static wallet metadata as defined in the spec:
https://openid.net/specs/openid-4-verifiable-presentations-1_0-20.html#name-a-set-of-static-configurati

If you need different wallet metadata downstream then you will need to construct it.

Reply:
Ah, okay, let me construct the metadata at instantiation.